### DISCUSSION TITLE
Issue with own E-Mail Template and Invitation Link

### DISCUSSION THREAD

**Discussion by DaveSH12** [dani-garcia/vaultwarden#4917] [Q&A] [answered]
Hello dear community,
I'm currently working on email templates and have copied the templates from the GitHub repository to /data/templates/email without making any changes to them yet. Now, when I invite a new user, they do receive the invitation email, but the link only leads directly to the server. The complete invitation URL is missing. When I disable the templates and it generates the email directly from src, it works fine. Does anyone have an idea what could be causing this?

Kind regards, David

**Comment by stefan0xC** [comment, marked as answer]
Which version are you using? You have probably downloaded the latest version of the templates which includes the changes from #4827 but if you are using the latest release (1.32.0) you would want the files for that version: https://github.com/dani-garcia/vaultwarden/tree/1.32.0/src/static/templates/email

**Reply by DaveSH12** [reply]
Thank you Stefan, you made my day
